src/pages/[platform]/build-a-backend/auth/connect-your-frontend/sign-in/index.mdx

```diff
@@ -1103,13 +1107,148 @@ func socialSignInWithWebUI() -> AnyCancellable {
 
 Your application's users can also sign in using passwordless methods. To learn more, visit the [concepts page for passwordless](/[platform]/build-a-backend/auth/concepts/passwordless/)
 
+<InlineFilter filters={["angular", "javascript", "nextjs", "react", "react-native", "vue"]}>
+
+### New auth flow type
+
+In order to facilitate the new passwordless sign in options, Cognito is introducing a new auth flow type known as `USER_AUTH`. This flow is designed to be flexible and supports both password and passwordless sign in factors.
+
+```ts
+const { nextStep } = await signIn({
+username: state.username,
+    options: {
+        authFlowType: "USER_AUTH",
+    }
+});
+```
+
+### Sign In with a Preferred Challenge for First Factor
```

Comment:
Also, be careful with headers, we created 3 new sections on this page. Should only be SMS OTP, Email OTP, WebAuthn Passkeys, and Password or SRP.

Reply:
Are we restricted to those 3? I think giving details on the new `USER_AUTH` flow is a necessary prerequisite to handle the new passwordless methods.

Reply:
We want to be consistent with the other libraries. We'd provide conceptual introduction to the auth flow on the `Switching Authentication Flows` page.

Reply:
A good compromise might be to add a `<Callout />` mentioning `USER_AUTH` with a link to that page?